ruma-appservice-api/Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2018"
 
 [dependencies]
 ruma-api = { version = "=0.17.0-alpha.4", path = "../ruma-api" }
+ruma-client-api = { version = "=0.10.0-alpha.3", path = "../ruma-client-api", features = ["client"] }
```

Comment:
I think this dependency (and the feature as a whole) should be behind a feature flag.

Reply:
How would you like the feature to be exposed in the `ruma` crate? As part of any of those three

https://github.com/ruma/ruma/blob/main/ruma/Cargo.toml#L44-L46

or as an additional feature?

Reply:
As an additional feature. Maybe it should also be prefixed `appservice-api`, or just `appservice`, but the exact name isn't that important, I'll adjust it before release if necessary.